README.md

```diff
@@ -44,13 +44,66 @@ The host must be connected to the Internet.
     $ docker-compose up
     ```
 
-If everything works fine, you can open a browser to visit the admin portal at http://reg.yourdomain.com . The default administrator username and password are admin/Harbor12345 .
-
-Create a new project, e.g. myproject, in the admin portal. You can then use docker commands to login and push images. The default port of Harbor registry server is 80:
-```sh
-$ docker login reg.yourdomain.com
-$ docker push reg.yourdomain.com/myproject/myrepo
-```
+  If everything works fine, you can open a browser to visit the admin portal at http://reg.yourdomain.com . The default administrator username and password are admin/Harbor12345 .
+
+  Create a new project, e.g. myproject, in the admin portal. You can then use docker commands to login and push images. The default port of Harbor registry server is 80:
+  ```sh
+  $ docker login reg.yourdomain.com
+  $ docker push reg.yourdomain.com/myproject/myrepo
+  ```
+
+4. Deploy harbor on kubernetes.
+  For now, it's a little tricky to start harbor on kubernetes because
+     1. registry uses https, so we need cert or workaround to avoid errors like this:
+        
+        ```
+        Error response from daemon: invalid registry endpoint https://{HOST}/v0/: unable to ping registry endpoint https://{HOST}/v0/
+        v2 ping attempt failed with error: Get https://{HOST}/v2/: EOF
+        v1 ping attempt failed with error: Get https://{HOST}/v1/_ping: EOF. If this private registry supports only HTTP or HTTPS with an unknown CA certificate, please add `--insecure-registry {HOST}` to the daemon's arguments. In the case of HTTPS, if you have access to the registry's CA certificate, no need for the flag; simply place the CA certificate at /etc/docker/certs.d/{HOST}/ca.crt
+        ```
+        
+        There is a workaround if you don't have a cert. The workaround is to add the host into the list of insecure registry by editting the ```/etc/default/docker``` file:
+        ```
+        sudo vi /etc/default/docker
+        ```
+        add the line at the end of file:
+        ```
+        DOCKER_OPTS="$DOCKER_OPTS --insecure-registry={HOST}"
+        ```
+        and restart docker service
+        ```
+        sudo service docker restart
+        ```
+     2. The registry config file need to know the IP (or DNS name) of the registry, but on kubernetes, you won't know the IP before the service is created. There are several workarounds to solve this problem for now:
```

Comment:
I'll add reference point about where it should be changed after checking in the config file.